Modules/Setup

```diff
@@ -134,9 +134,6 @@ faulthandler faulthandler.c
 # can call _PyTraceMalloc_NewReference().
 _tracemalloc _tracemalloc.c
 
-# PEG-based parser module -- slated to be *the* parser
-_peg_parser _peg_parser.c
```

Comment:
Why delete this? It serves a purpose for our timing tools.

Reply:
IIRC, we talked about removing this with the old parser as its main purpose was to be able to interface with both parsers (with some extra options), I think it will be simpler if we just adapt our scripts (in a different PR) to just use `ast.parse` or similar for timings of the new parser. The overhead of constructing the AST will be common on every run, so you could do still the same things we could do before.

Reply:
Okay.